### PR TITLE
Api: reminder cron

### LINKED ISSUE #397: Cron for reminder and room creation

### PR DESCRIPTION
**Pull request by HugoGresse**
- implement reminder cron, sent every 5 minutes at fixed minutes

Fix #397

### CODE REVIEW

File: functions/src/cron/remindersCron.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        pushsSentTotal += pushsSent.length
+    }
+
+    if (emailsSentTotal > 0 || smsSentTotal > 0 || pushsSentTotal > 0) {
+        console.info(
+            `Reminder cron sent: email:${emailsSentTotal}, sms:${smsSentTotal}, push:${pushsSentTotal}`
+        )
+    }
+
+    return
```
Comment (ntchou) [inline functions/src/cron/remindersCron.ts:64]: necessary?

**Comment by HugoGresse** [comment]
missing part: the reminder content is not saying anything about at which time the meeting is, refer to this: 
```

Bonjour ! :sourire_yeux_rieurs: Retrouvez {FullName.Invitant} pour la conversation vidéo prévue à xxhxx, le XX/XX sur ce lien: {https://InstantVisio.com} :réveil:
```

File: functions/src/cron/remindersCron.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        await ReminderDao.update({
+            reminderId: reminder.id,
+            isSent: true,
+        })
+        emailsSentTotal += emailsSent.length
+        smsSentTotal += smssSent.length
+        pushsSentTotal += pushsSent.length
+    }
+
+    if (emailsSentTotal > 0 || smsSentTotal > 0 || pushsSentTotal > 0) {
```
Comment (MattiaPrimavera) [inline functions/src/cron/remindersCron.ts:58]: `> 0` not necessary

File: functions/src/cron/test.http [hunk 3]
```diff
@@ -0,0 +1 @@
+GET http://localhost:5000/instantvisio-dev/us-central1/api/api/v1-tests/reminders/runCron
```
Comment (MattiaPrimavera) [inline functions/src/cron/test.http:1]: are you following a pattern for these `.http` files ?

**Review by MattiaPrimavera — APPROVED** [review]

**Comment by MattiaPrimavera** [comment]
Good job man! Some conflicts to be solved

**Review by ntchou — APPROVED** [review]

File: functions/src/cron/test.http (hunk 3 above)
Comment (HugoGresse) [reply functions/src/cron/test.http:1]: yes, it directly integrated in mY IDE so I can run http request in the same window

Comment (HugoGresse) [reply functions/src/cron/test.http:1]: this one will be removed before merge though

**Comment by HugoGresse** [comment]
I've pushed some more improvement. The Date is formatted but not based on the user local and timezone, this will be done in another PR using Luxon

**Pull request merged**